vulture_os/toolkit/api_parser/beyondtrust_pra/beyondtrust_pra.py

```diff
@@ -0,0 +1,278 @@
+#!/home/vlt-os/env/bin/python
+"""This file is part of Vulture OS.
+
+Vulture OS is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Vulture OS is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Vulture OS.  If not, see http://www.gnu.org/licenses/.
+"""
+__author__ = "Nicolas Lançon"
+__credits__ = []
+__license__ = "GPLv3"
+__version__ = "4.0.0"
+__maintainer__ = "Vulture OS"
+__email__ = "contact@vultureproject.org"
+__doc__ = 'Beyondtrust PRA API'
+__parser__ = 'Beyondtrust PRA'
+
+import base64
+import json
+import logging
+import time
+
+import requests
+import xmltodict
```

Comment:
use defusedxml instead

Reply:
xmltodict is more efficient here

Reply:
Checked yesterday with team : xmltodict is more secure than defusedxml (by Snyk Advisor)